### PR TITLE
Jalmogo/add mississippi flavor

### PR DESCRIPTION
**Pull request by jalMogo**
currently on staging

### CODE REVIEW

File: src/flavors/mississippi/config.json [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+        "type": "raster",
+        "tiles": [
+          "https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}"
+        ],
+        "tileSize": 256
+      },
+      "nitrogen": {
+        "type": "raster",
+        "tiles": [
+          "https://ecotopia.today/Mississippi/Nitrogen/{z}/{x}/{y}.png"
```
Comment (jalMogo) [inline src/flavors/mississippi/config.json]: I'm not sure why, but this layer isn't appearing. How would we debug this?

File: src/flavors/mississippi/config.json [hunk 2]
```diff
@@ -0,0 +1,799 @@
...
+            "id": "nitrogen-plants-labels",
+            "type": "circle",
+            "source": "nitrogen-plants",
+            "source-layer": "nitrogen-plants",
+            "paint": {
+              "circle-radius": {
+                "base": 1.75,
+                "stops": [[12, 2], [22, 180]]
+              },
+              "circle-color": "#fbb03b"
```
Comment (jalMogo) [inline src/flavors/mississippi/config.json]: I'm trying to draw this geojson source as circles, but they don't appear on the map. @goldpbear any suggestions on how to debug this?

Comment (goldpbear) [reply src/flavors/mississippi/config.json]: I think removing the `source-layer` line at line 259 will fix this. `source-layer` is only needed for layers rendering vector tile sources.

In general, Mapbox logs pretty good debugging information. There are cases, though, (like here) where Mapbox seems to swallow misconfigured layers and just won't render them. I've worked through a few situations like that so I might be able to help debug layers that seemingly won't render.

File: src/flavors/mississippi/config.json (hunk 1 above)
Comment (goldpbear) [reply src/flavors/mississippi/config.json]: I'm getting 404s on all the tile requests for this source, e.g.: https://ecotopia.today/Mississippi/Nitrogen/4/3/6.png.

Is the source info correct?

File: src/flavors/mississippi/config.json (hunk 2 above)
Comment (jalMogo) [reply src/flavors/mississippi/config.json]: Haha, yes, that did the trick!

Thanks for the tip :+1:

**Pull request merged**